LBHFSSPublicAPI/V1/Infrastructure/DatabaseContext.cs

```diff
@@ -259,6 +260,25 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
                     .HasConstraintName("service_taxonomies_taxonomy_id_fkey");
             });
 
+            modelBuilder.Entity<AnalyticsEvent>(entity =>
+            {
+                entity.ToTable("service_analytics");
+
+                entity.Property(e => e.Id)
+                    .HasColumnName("id")
+                    .UseIdentityAlwaysColumn(); ;
+
+                entity.Property(e => e.TimeStamp).HasColumnName("description");
```

Comment:
Probably failing to understand this but should the `.HasColumnName` be "timestamp" or similar rather than "description"?